.github/workflows/lint_0.yml

```diff
@@ -23,10 +23,10 @@ jobs:
       - name: Checkout repo @ SHA - ${{ github.sha }}
         uses: actions/checkout@v4
 
-      - name: Set up Python 3.12
```

Comment:
https://github.com/pylint-dev/pylint/issues/10112

Reply:
Disable for now. If approved, I can create an issue to track and remove this when the bug gets fixed

Reply:
Can we add a comment to bump it here after the bug is fixed instead of adding pylint ignore comments to the code?

Reply:
Yes. This could be another option as well.

Reply:
Thanks!